### DISCUSSION TITLE
Multiple regions

### DISCUSSION THREAD

**Discussion by robkallen** [planetscale/discussion#347] [Q&A] [answered]
I was wondering if planetscale could handle a use case I have.  We have a multi-tenant SaaS app where our client data is co-mingled in the same database (not planetscale), hosted in a US region.  However, we have one client that is requesting their data be stored in an EU region.  With that in mind is it possible with planetscale to have two instances of the same database in different regions with different data, but with the same schema?

**Comment by orware** [comment, marked as answer]
Hi @robkallen,

There may be other considerations that are also required in such a situation, but I don't see any reason why you wouldn't be able to achieve the data separation goal you've described.

It sounds like the logic itself to direct the client's database requests to a different database located within the EU would need to be added within the application itself, but if I'm understanding the rest of what you've described correctly the rest should be doable within PlanetScale too.

The main assumption I am making here is that essentially you would have one database that just contains the one client's data, or any EU user data, and all of that would be separate from your other database hosted in the US.

In this situation, you could approach this in two different ways: either creating this separation at the database level, or possibly at the branch level, within PlanetScale.

If you separated things at the database level you might end up with two databases like `us-prod` and `eu-prod` and each of those might have their own primary `main` branch. This would give you data separation, along with the logical database separation too within our UI. Schema-wise, you would primarily need to apply the same schema changes to both of these separate database branches whenever those crop up via our [deploy request process](https://planetscale.com/docs/concepts/deploy-requests).

The other approach could be to keep things within a single database, which I'll call `prod`, but on our [Scaler plan](https://planetscale.com/docs/concepts/billing#planetscale-plans) you have the option of creating two production branches. In this case, you could create one for the US and one for the EU, for example `us-main` and `eu-main`, and place them in the appropriate region. The data in each of those branches would be independent of the other, and as in the previous example, for any schema changes you would need to apply them to both of these branches separately.

I hope this helps to explain how you might be able to achieve what you've described above!

**Comment by tafelnl** [comment]
@robkallen Did you ever end up implementing a (similar) thing as what you described? I'm curious as to what steps you took in order to achieve such a thing. Because we have a similar use case
